libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py

```diff
@@ -174,9 +175,35 @@ def _convert_prompt_msg_params(
         self,
         messages: List[BaseMessage],
         **kwargs: Any,
-    ) -> dict:
+    ):
+        """
+        Converts a list of messages into a dictionary containing the message content
+        and default parameters.
+
+        Args:
+            messages (List[BaseMessage]): The list of messages.
+            **kwargs (Any): Optional arguments to add additional parameters to the
+            resulting dictionary.
+
+        Returns:
+            Dict[str, Any]: A dictionary containing the message content and default
+            parameters.
+
+        """
+        messages_dict = {
+            "messages": [
+                convert_message_to_dict(m)
```

Comment:
can we just update this method

Reply:
That might be a difference between Qianfan API and Open-ai API. In Qianfan API, there is a field called `system`, which has the same level with `messages`. For example:

```
{
    "messages":[
        {
            "role":"user",
            "content":"Who are you?"
        }
    ],
    "system":"You are an artificial intelligence assistant. You were developed by Baidu Company, and you can provide assistance, advice, and chat ability to users. "
}
```

Reply:
> That might be a difference between Qianfan API and Open-ai API. In Qianfan API, there is a field called `system`, which has the same level with `messages`. For example:
> 
> ```
> {
>     "messages":[
>         {
>             "role":"user",
>             "content":"Who are you?"
>         }
>     ],
>     "system":"You are an artificial intelligence assistant. You were developed by Baidu Company, and you can provide assistance, advice, and chat ability to users. "
> }
> ```
> 
> So only updating the method `convert_message_to_dict` might be difficult. @baskaryan FYI

CC, if that OK for merge? @baskaryan

Reply:
makes sense, thank you @NuODaniel!